backend/app/ingestion/phase2/IngestStorePolling.scala

```diff
@@ -71,12 +72,15 @@ class IngestStorePolling(
               val result = processKey(key)
               result match {
                 case Left(failure) =>
-                  failureToResultMapper.failureToResult(failure)
+                  metricsService.updateCloudwatchMetric(Metrics.itemsFailed)
                   logger.warn(s"Failed to process $key: $failure")
                 case _ => ingestStorage.delete(key)
               }
               result
             }.collect { case Right(success) => success }
+            metricsService.updateCloudwatchMetrics(List(
+              MetricUpdate(Metrics.itemsIngested, results.size),
+              MetricUpdate(Metrics.batchesIngested, 1)))
```

Comment:
A bit confused at this number. Should it be the number of batches ingested as part of the overall dataset? I'm not sure the worker has the information to know - feel free to ignore.

Reply:
ah the '1' is the number that failed and I think in this case that's just a single batch that's failed?